flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/internal/options/JdbcOptions.java

```diff
@@ -150,7 +168,7 @@ public JdbcOptions build() {
 				});
 			}
 
-			return new JdbcOptions(dbURL, tableName, driverName, username, password, dialect);
+			return new JdbcOptions(dbURL, tableName, driverName, username, password, dialect,connectionCheckTimeoutSeconds);
```

Comment:
Add space before `connectionCheckTimeoutSeconds`.

Reply:
thanks , review  my code. i had done with your suggestion .But i dont konw how to get connectionCheckTimeoutSeconds in JdbcBatchingOutputFormat.java  ,so i remove it .